tox_poetry_installer/utilities.py

```diff
@@ -182,7 +182,7 @@ def find_project_deps(
     venv: "_poetry.VirtualEnv",
     poetry: "_poetry.Poetry",
     extras: Sequence[str] = (),
-) -> List[PoetryPackage]:
+) -> Set[PoetryPackage]:
```

Comment:
We can't return dependency sequences as sets because python sets are unordered and we need dependencies to be installed in the specified order. The `identify_transients()` function does manual de-dup [here](https://github.com/enpaul/tox-poetry-installer/blob/0.8.5/tox_poetry_installer/utilities.py#L130)

For additional context, #41 is where the problem with using sets was first reported